cmds/portmaster-start/run.go

```diff
@@ -121,6 +121,20 @@ func getExecArgs(opts *Options, cmdArgs []string) []string {
 	if stdinSignals {
 		args = append(args, "--input-signals")
 	}
+
+	// see https://www.freedesktop.org/software/systemd/man/pam_systemd.html#type=
```

Comment:
Does that mean this this only works when systemd is used?

Reply:
that environment variable is populated by systemd. I did not find any other reasonable way of detecting if wayland is used (and not just available or running for some other session). Most systems are using systemd anyway so this fix is for them. Non-systemd users can still workaround the issue by appending the args manually as suggested in  https://github.com/safing/portmaster/issues/495

Reply:
Though, for example GNOME also depends on that environment variable to decide in which mode to run so relying on that should be reasonable